data/strings.yml

```diff
@@ -120,6 +120,7 @@ workflows:
     details: Are you sure you want to delete the <workflow_id> workflow?
 code_signing:
   available_project_types_load: Loading, wait a sec...
+  expose_for_pr_popover_content: You can decide which files should be exposed for / available in Pull Request builds. <br> Be careful, exposing a secret is a potential security risk.
```

Comment:
Instead of *secret*, shouldn't it be something like *private file* or something? @viktorbenei so they won't confuse it with the secret env vars.

Also, remove whitespace before `<br>`